metadata-ingestion/src/datahub/ingestion/source/aws/aws_common.py

```diff
@@ -89,7 +90,7 @@ def get_session(self) -> Session:
                 region_name=self.aws_region,
             )
         else:
-            return Session(region_name=self.aws_region)
+            return Session(region_name=self.aws_region, profile_name=self.aws_profile)
```

Comment:
Is the `profile_name` param supported by our min version of boto3?

Reply:
it is there for 7 years, so I think we should be fine